extensions/gdx-setup/src/com/badlogic/gdx/setup/resources/android/res/values/color.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="white">#FFFFFFFF</color>
```

Comment:
Do not name it "white", but "ic_background_color" to prevent misunderstandings

Reply:
Thank you, good point, I'll make the change.